modules/vsphere-problem-detector-running.adoc

```diff
@@ -34,22 +34,6 @@ $ oc wait pods -l name=vsphere-problem-detector-operator \
     --for=delete --timeout=5m -n openshift-cluster-storage-operator
 ----
```

Comment:
It's better to remove this part, as the vsphere-problem-detector-operator is managed, the pod will be scaled very quickly and you will probably wait till timeout.
Suggest to use alternative way:

The vsphere-problem-detector-operator will be scaled up automatically, so you can run the following command to check the AGE of pod to see if it is restarted:
```
oc -n openshift-cluster-storage-operator get pod -l name=vsphere-problem-detector-operator

NAME                                                 READY   STATUS    RESTARTS   AGE
vsphere-problem-detector-operator-77486bd645-9ntpb   1/1     Running   0          11s
```

Reply:
oh I see. Is reducing the timeout an option? But I do see why the alternative you provided is more efficient.

Reply:
How about having both commands?

Reply:
The pod will start up very very quickly, so you probably have no chance to see the deletion of the old one, you will already get a new one when you execute the "oc wait" cmd and it doesn't work (just timeout). 
If you want to continuously check the status, we can add "w" in my command, I don't think the previous cmd works.
```
oc -n openshift-cluster-storage-operator get pod -l name=vsphere-problem-detector-operator -w

NAME                                                 READY   STATUS    RESTARTS   AGE
vsphere-problem-detector-operator-77486bd645-9ntpb   1/1     Running   0          11s
```

Reply:
@duanwei33 I have incorporated your suggestions. Can you review if it looks fine?